datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/dashboard.py

```diff
@@ -66,13 +66,14 @@ def export(ctx, url, integration):
         abort(str(e).replace(api_key, '*' * len(api_key)).replace(app_key, '*' * len(app_key)))
 
     payload = response.json()
-    payload.setdefault('author_info', {})
-    payload['author_info']['author_name'] = 'Datadog'
-    payload.setdefault('created_by', {})
-    payload['created_by']['email'] = 'support@datadoghq.com'
-    payload['created_by']['handle'] = 'support@datadoghq.com'
-    payload['created_by']['name'] = 'Datadog'
-    payload['created_by'].pop('icon', None)
+    payload.pop('author_info', None)
```

Comment:
We don't need this field anymore? specifically the fields that point to Datadog support info

https://github.com/DataDog/integrations-core/blob/a947e71101e8cb830eb4e7aa01da62ba886da185/cilium/assets/dashboards/overview.json#L1-L16

Reply:
No, they don't seem to be needed when creating an integration